lxd/resources/network.go

```diff
@@ -605,54 +606,106 @@ func GetNetworkState(name string) (*api.NetworkState, error) {
 		network.Bond = &bonding
 	}
 
-	// Populate bridge details.
-	bridgePath := fmt.Sprintf("/sys/class/net/%s/bridge", name)
-	if sysfsExists(bridgePath) {
-		bridge := api.NetworkStateBridge{}
+	// Populate bridge details
+	ovs := openvswitch.NewOVS()
+	exists := false
+	if ovs.Installed() {
+		exists, err = ovs.BridgeExists(name)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	bridge := api.NetworkStateBridge{}
+
+	if !exists {
```

Comment:
`exists` is a confusing variable name. Can you please call it something like `isOVSBridge`?